marketplaceItems/sitPoint/appResources/appData/sitClient.js

```diff
@@ -0,0 +1,904 @@
+//
+// sit.js
+//
+// Created by Clement Brisset on 3/3/17
+// Modified by Robin Wilson, Rebecca Stankus, and Alexia Mandeville January 2019
+//
+// Copyright 2019 High Fidelity, Inc.
+//
+// Distributed under the Apache License, Version 2.0.
+// See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+// Sit setup:
+// 
+// Best added to a stationary chair or entity. Center of seat is indicated by the entity's registration point.
+// Ex: a stool for an avatar to sit in the middle on top, registrationPoint = { x: 0.5, y: 1.0, z: 0.5 }.
+// 
+// Add sit.js and sitServer.js to an entity
+// Ensure both are running.
+//
+// Configurable variables are:
+// - SHOW_PRESIT_OVERLAY_IN_HMD
+// - CHAIR_OFFSET_RATIO
+// 
+
+/* globals Entities, Script, AnimationCache, Settings, MyAvatar, DriveKeys, AvatarList,
+ Vec3, HMD, Overlays, Camera, isInEditMode */
+
+(function () {
+
+    // Configurable Variables
+
+    // true - shows "Face Forward Overlay" before sit; false - sits immediately with no overlay
+    var SHOW_PRESIT_OVERLAY_IN_HMD = true;
+
+    // Used to calculate pin hip position. Adds CHAIR_OFFSET_RATIO * chair's y dimension to the y center of the seat.
+    // Avatar sit position y = CHAIR_OFFSET_RATIO * height of chair
+    var CHAIR_OFFSET_RATIO = 0.1;
+
+    Script.include("/~/system/libraries/utils.js");
+
+    var entityID = null;
+
+    var SETTING_KEY = "com.highfidelity.avatar.isSitting";
+
+    var SIT_SETTLE_TIME = 350; // ms
+
+    var SITTING_DEBUG = false; // turn on sitting debug print statements
+    var OVERLAY_DEBUG = false; // turn on overlay debug print statements
+
+    var ONE_HUNDRED_AND_TWENTY_PERCENT = 1.2;
```

Comment:
I understand why you're doing this. However, this variable name gives no context to the reader. There's only one place you're using this value. Perhaps consider changing this to `MAX_HEAD_DEVIATION_RATIO` or something similarly more descriptive? Apply that same line of thinking to the rest of these `var` names.